apps/common/views/error.html

```diff
@@ -3,11 +3,9 @@
     {{#t}}error.header.problem{{/t}}
   {{/header}}
   {{$content}}
-    <p>{{#t}}error.content.problem.p1{{/t}}</p>
-    {{#showStack}}
```

Comment:
can you re-add the whole `{{#showStack}}` block, at the end of the `.error` div? The stack is hidden/shown based on the environment, so we can see full stacktrace error messages in dev/uat/branch but not in staging/prod.